lib/logdna/writer.go

```diff
@@ -127,10 +131,22 @@ func extractToken(u *url.URL) (token string, err error) {
 	return
 }
 
+func extractPEN(u *url.URL) (pen string) {
+	if len(pen) == 0 {
+		pen = "48950"
+	}
+
+	return
+}
```

Comment:
I'm confused about this, does it simply return a constant all the time? `pen` as a return value will always pass `len(pen) == 0`.